packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -384,7 +384,7 @@ export class CeramicCliUtils {
     const id = StreamID.fromString(streamId)
 
     await CeramicCliUtils._runWithCeramicClient(async (ceramic: CeramicApi) => {
-      const result = await ceramic.pin.add(id)
+      const result = await ceramic.admin.pin.add(id)
```

Comment:
Not sure if we really intended to put all pin apis behind admin api, but I think it makes sense to make this symmetrical. Streams are pinned by default on creation anyway so it shouldn't be a problem.